### PR TITLE
Reduces the space between timeline events

### LINKED ISSUE #911: Shorten the distance between Posts on Release Notes + What’s new.
This is minor tweak to reduce the margin between items posted on the what's new and release notes timelines

Screen Shot 2021-02-18 at 5 25 44 PM

### PR DESCRIPTION
**Pull request by caylahamann**
Reduces the space between timeline events for whatsnew and releasenotes 

closes #911 

## Before 
Screen Shot 2021-03-04 at 2 41 56 PM

## After
Screen Shot 2021-03-04 at 2 49 24 PM

### CODE REVIEW

**Comment by jerelmiller** [comment]
Could we instead use `2rem`? While its not a written rule right now, I've generally been using `2rem` to be the space between 2 groups of content together. This seems to work pretty well site-wide. Would be good to get this into more of a formal "spacing" guide at some point 😄

**Comment by caylahamann** [comment]
@jerelmiller yep! that's a good call. I'll make that update.

**Comment by caylahamann** [comment]
@jerelmiller updated as well as updated the PR with the new photo :) I'll wait on feedback from mark and austin

**Review by jerelmiller — APPROVED** [review]
Assuming others like it, this is much better! Thank you 🙂

**Comment by jpvajda** [comment]
LGTM!

**Comment by caylahamann** [comment]
I'm going to merge this in! If mark doesn't like it we can always change it to a different value, it's a pretty small change :)

**Pull request merged**
